Add webhook abuse section (#797)
clerk/clerk-docs: docs/integrations/webhooks/overview.mdx

  alt="The Webhooks page of the Clerk Dashboard. The Replay menu, for replaying failling or missing webhook messages, is open and showing three options."
/>

## Sync data to your database

You can find a guide on how to use webhooks to sync your data to your database [here](/docs/integrations/webhooks/sync-data).

## Protecting your webhooks from abuse

> If you'd like to ensure that the api route receiving the webhook can only be hit by your own app, there are a few protections you can put in place.

Reviewer: ```suggestion
To ensure that the api route receiving the webhook can only be hit by your app, there are a few protections you can put in place:
```

> First, each webhook request sent by Svix is [signed](https://www.wikiwand.com/en/Digital_signature), and you can verify the signature to ensure that it was sent by Svix, rather than someone else trying to spam or attack your endpoint. Check out Svix's guide on [how to verify webhooks with the svix libraries](https://docs.svix.com/receiving/verifying-payloads/how) or [how to verify webooks manually](https://docs.svix.com/receiving/verifying-payloads/how-manual) for more information on how to verify the webhook signature.

Reviewer: ```suggestion
- **Verify the request signature**: Svix webhook requests are [signed](https://www.wikiwand.com/en/Digital_signature) and can be verified to ensure the request is not malicious. To learn more, check out Svix's guide on [how to verify webhooks with the svix libraries](https://docs.svix.com/receiving/verifying-payloads/how) or [how to verify webooks manually](https://docs.svix.com/receiving/verifying-payloads/how-manual).
```
Just want to make this more succinct

> 

Reviewer: ```suggestion
```

Deleting this line to connect the bulleted list
